python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -67,7 +67,12 @@ class RandomForestClassifier(BaseRandomForestModel, DelayedPredictionMixin,
     n_estimators : int (default = 10)
                    total number of trees in the forest (not per-worker)
     handle : cuml.Handle
-        If it is None, a new one is created just for this class.
+        Specifies the cuml.handle that holds internal CUDA state for
```

Comment:
Can we automate the splicing in of this handle docstring with one of the docstring generators @dantegd added?

Reply:
Yes but the design is a little more complicated than the `@generate_docstring()` decorators. To make release, I decided a find/replace all would be quicker than designing a complicated docstring generator. Additionally, I found some other tools that may work better for this situation.

Reply:
what other tools? It would not be a complicated decorator, it would be very similar to the existing insert into docstring, no?